src/power/config.js

```diff
@@ -6,7 +6,7 @@ const TESTS = [
   {
     id: 'cpu',
     label: 'CPU',
-    filter: { eq: { test: 'cpu' } },
+    filter: { eq: { test: 'power-cpu' } },
```

Comment:
Will this also show the historical data? Perhaps this could be an 'or', or a suffix match?

Reply:
@davehunt is correct

```
{"or":[
    { eq: { test: 'cpu' } ,
    { eq: { test: 'power-cpu' } }
]}
```

or

```
{ in: { test: ['cpu', 'power-cpu' } }
```

Reply:
Thanks for pointing that out :)
I changed it to show historical data also:
![health-power-historical](https://user-images.githubusercontent.com/51707553/64098088-a895d880-cd6e-11e9-8a0f-eb37009c0dda.png)